config/initializers/languages.rb

```diff
@@ -10,38 +10,38 @@
 
 LANGUAGE_NAMES = {
   'ar' => 'العربية',
-  'bg' => 'български',
+  'bg' => 'Български',
   'ca' => 'Català',
-  'cs' => 'čeština',
-  'da' => 'dansk',
+  'cs' => 'Čeština',
+  'da' => 'Dansk',
   'de' => 'Deutsch',
-  'el' => 'ελληνικά',
+  'el' => 'Ελληνικά',
   'en' => 'English',
-  'es' => 'español',
-  'fa' => 'زبان فارسی',
-  'fi' => 'suomi',
-  'fr' => 'français',
+  'es' => 'Español',
+  'fa' => 'فارسی',
+  'fi' => 'Suomi',
+  'fr' => 'Français',
   'gl' => 'Galego',
   'hi' => 'हिन्दी',
-  'hu' => 'magyar',
+  'hu' => 'Magyar',
   'id' => 'Bahasa Indonesia',
-  'it' => 'italiano',
+  'it' => 'Italiano',
   'ja' => '日本語',
   'km' => 'ភាសាខ្មែរ',
-  'ko' => '한국어 [韓國語]',
-  'lt' => 'lietuvių kalba',
+  'ko' => '한국어',
+  'lt' => 'Lietuvių',
   'nb' => 'Bokmål',
   'nl' => 'Nederlands',
-  'nn' => 'Norsk',
-  'pl' => 'polski',
-  'pt_BR' => 'português',
-  'ro' => 'român',
-  'ru' => 'Русский язык',
-  'sk' => 'slovenčina',
+  'nn' => 'Nynorsk',
```

Comment:
That's interesting. Apparently the proper locale name for `Norsk` is `no`, `nn` is `Nynorsk` -- which are different things.